tobago-example/pom.xml

```diff
@@ -420,11 +420,11 @@
           <scope>compile</scope>
         </dependency>
         <!-- this enables the development mode -->
-        <dependency>
+        <!--dependency>
           <groupId>org.apache.myfaces.tobago</groupId>
           <artifactId>tobago-config-dev</artifactId>
           <version>${tobago.version}</version>
-        </dependency>
+        </dependency-->
       </dependencies>
```

Comment:
tobago-config-dev should be removed. It could be activated with the 'dev' profile.

Reply:
removed